syft/cataloging/unknowns.go

```diff
@@ -0,0 +1,15 @@
+package cataloging
+
+type UnknownsConfig struct {
+	RemoveWhenPackagesDefined         bool
+	IncludeExecutablesWithoutPackages bool
+	IncludeUnexpandedArchives         bool
```

Comment:
These 3 options are part of the public API and should be looked at perhaps with more scrutiny than other aspects of this PR